src/View/Helper/HtmlHelper.php

```diff
@@ -5,6 +5,12 @@ class HtmlHelper extends \Cake\View\Helper\HtmlHelper
 {
     use OptionsAwareTrait;
 
+    private $grid = [];
+
+    private $gridOptions = [];
+
+    private $gridCounter = 0;
```

Comment:
Does it have to be private? Often times people extend this HtmlHelper and would have to redeclare private stuff.
Protected should suffice IMO.